# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1548377405 28800
#      Thu Jan 24 16:50:05 2019 -0800
# Node ID 2f77d977606aea8e84e8ff49287a0378af8c8ec3
# Parent  fb100b270550bb3b8f112654f4e2defae44f538b
* NEWS: Refer to version "6" instead of "6.0".

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -1,49 +1,49 @@
-Summary of important user-visible changes for version 6.0 (yyyy-mm-dd):
+Summary of important user-visible changes for version 6 (yyyy-mm-dd):
 ----------------------------------------------------------------------
 
- ** New functions added in 6.0:
+ ** New functions added in Octave 6:
 
       lightangle
       verLessThan
 
  ** Deprecated functions.
 
-    The following functions have been deprecated in Octave 6.0 and will
+    The following functions have been deprecated in Octave 6 and will
     be removed from Octave 8 (or whatever version is the second major
-    release after 6.0):
+    release after 6):
 
       Function               | Replacement
       -----------------------|------------------
       output_max_field_width | output_precision
 
  ** The following functions were deprecated in Octave 4.4 and have been
-    removed from Octave 6.0.
+    removed from Octave 6.
 
       Function             | Replacement
       ---------------------|------------------
       chop                 | sprintf for visual results
       desktop              | isguirunning
       tmpnam               | tempname
       toascii              | double
       java2mat             | __java2mat__
 
  ** Deprecated graphics properties.
 
     The following properties or allowed corresponding values have been
-    deprecated in Octave 6.0 and will be removed from Octave 8 (or
-    whatever version is the second major release after 6.0):
+    deprecated in Octave 6 and will be removed from Octave 8 (or
+    whatever version is the second major release after 6):
 
       Object               | Property                | Value
       ---------------------|-------------------------|-------------------
 
 
  ** The following properties or allowed corresponding values were
-    deprecated in Octave 4.4 and have been removed from Octave 6.0:
+    deprecated in Octave 4.4 and have been removed from Octave 6:
 
       Object               | Property                | Value
       ---------------------|-------------------------|-------------------
       figure               | doublebuffer            |
                            | mincolormap             |
                            | wvisual                 |
                            | wvisualmode             |
                            | xdisplay                |
